src/main/java/com/marklogic/contentpump/ThreadManager.java

```diff
@@ -126,6 +127,10 @@ public void parseCmdlineOptions(CommandLine cmdline, Command cmd) {
      */
     public void queryServerMaxThreads(ContentSource cs)
         throws IOException {
+        if (threadCount != 0) {
```

Comment:
What's this check for, to prevent queryServerMaxThread() from being called simultaneously more than once (but I thought scheduleWithFixedDelay() fixes that?  I missing something